web/src/components/PreflightResultPage.tsx

```diff
@@ -304,8 +339,9 @@ class PreflightResultPage extends Component {
         this.state.getKotsPreflightResultJob.stop();
         this.setState({ preflightResultCheckCount: 0 });
       }
-      let parsedStatusResults = {};
+      let parsedStatusResults: PreflightProgress | null = null;
       try {
+        // TODO: this is a nested JSON in JSON- should refactor backend
```

Comment:
had to run the app to create this type since in the backend it's just a "string"